x/token/keeper/msg_server_test.go

```diff
@@ -1,8 +1,10 @@
 package keeper_test
 
 import (
+	"encoding/json"
 	"fmt"
 	abci "github.com/tendermint/tendermint/abci/types"
+	"strings"
```

Comment:
Please separate `tendermint` package from go default package to follow go recommended spec.

Reply:
Coule you tell me what imports style I should use?
There seems some options showing different import styles. Is it enough to separate `tendermint` from `stdlib`?

<img width="737" alt="스크린샷 2023-09-26 오전 11 38 26" src="https://github.com/Finschia/finschia-sdk/assets/119839167/81625f18-8bc4-4082-b053-f4fb371102f9">

Reply:
<img width="733" alt="image" src="https://github.com/Finschia/finschia-sdk/assets/41176085/2510cc86-9d37-4460-aeb0-2b0759b974ea">

Reply:
Thank you.